plugins/services/src/js/components/ServiceStatusIcon.tsx

```diff
@@ -19,7 +19,7 @@ import ServiceTree from "../structs/ServiceTree";
 
 const UNABLE_TO_LAUNCH_TIMEOUT = 1000 * 60 * 30; // 30 minutes
 
-const getTooltipContent = (service: Service, content: JSX.Element) => {
+const getTooltipContent = (service: Service | Pod, content: JSX.Element) => {
   const servicePath = encodeURIComponent(service.getId());
```

Comment:
Is this change for this PR?

Reply:
The first commit is from #3812, now that it’s merged I’ll rebase this.